playbooks/nomad.yml

```diff
@@ -0,0 +1,25 @@
+---
+
+- name: Set up Nomad server
+  hosts: nomad-server
+  become: true
+  roles:
+    # Note: we don't run the common-server role because our Nomad servers
+    # are on the load balancing servers, so running it would cause issues
+    # like the hostname changing every now and then.
+    # We can technically skip such tasks with variables, but we don't need
+    # anything specific out of common-server that is tailored for Nomad.
```

Comment:
Maybe we should just run the common-server role for `hosts: all`?  However, this would make it more difficult to use the individual playbooks, so it's probably not a good idea.

Reply:
Yeah I think that may cause issues we can't foresee right now, so I'll leave it as is for now.